projects/schematics/src/migrations/5_0/constructor-deprecations/data/constructor-deprecations.migration.ts

```diff
@@ -0,0 +1,4762 @@
+      },
+      {
+        className: 'Injector',
+        importPath: '@angular/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'CmsComponentsService',
+        importPath: '@spartacus/storefront',
+      },
+      {
+        className: 'UnifiedInjector',
+        importPath: '@spartacus/core',
+      },
+    ],
+  },
+  {
+    class: 'ConsignmentTrackingComponent',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'UserOrderService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ModalService',
+        importPath: '@spartacus/storefront',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'UserOrderService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ModalService',
+        importPath: '@spartacus/storefront',
+      },
+    ],
+    addParams: [
+      {
+        className: 'OrderHistoryFacade',
+        importPath: '@spartacus/order/root',
+      },
+      {
+        className: 'ModalService',
+        importPath: '@spartacus/storefront',
+      },
+    ],
+  },
+  {
+    class: 'MiniCartComponent',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'MiniCartComponentService',
+        importPath: '@spartacus/cart/base/components/mini-cart',
+      },
+    ],
+  },
+  {
+    class: 'NavigationUIComponent',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'Router',
+        importPath: '@angular/router',
+      },
+      {
+        className: 'Renderer2',
+        importPath: '@angular/core',
+      },
+      {
+        className: 'ElementRef',
+        importPath: '@angular/core',
+      },
+      {
+        className: 'HamburgerMenuService',
+        importPath: '@spartacus/storefront',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'Router',
+        importPath: '@angular/router',
+      },
+      {
+        className: 'Renderer2',
+        importPath: '@angular/core',
+      },
+      {
+        className: 'ElementRef',
+        importPath: '@angular/core',
+      },
+      {
+        className: 'HamburgerMenuService',
+        importPath: '@spartacus/storefront',
+      },
+    ],
+    addParams: [
+      {
+        className: 'Router',
+        importPath: '@angular/router',
+      },
+      {
+        className: 'Renderer2',
+        importPath: '@angular/core',
+      },
+      {
+        className: 'ElementRef',
+        importPath: '@angular/core',
+      },
+      {
+        className: 'HamburgerMenuService',
+        importPath: '@spartacus/storefront',
+      },
+      {
+        className: 'WindowRef',
```

Comment:
now, `removeParams` is the same as `deprecatedParams`. but, for this case, it only add one param `WindowRef`.